configs/katello/4.9.yaml

```diff
@@ -0,0 +1,80 @@
+---
+:project: katello
+:github_org: katello
+:nightly: false
+:mash_scripts:
+  - katello-4.9-el8
+  - katello-candlepin-4.9-el8
+:tags:
+  - name: katello-candlepin-4.9-el8
+    based_off: katello-candlepin-4.9-el8
+    build_target: katello-4.9-el8-build
+    arches:
+      - x86_64
+  - name: katello-4.9-el8
+    based_off: katello-4.9-el8
+    helper_tags:
+      katello-thirdparty-el8:
+      katello-4.9-el8-override:
+    build_target: katello-4.9-el8-build
+    build_package_group_source_tag: katello-4.9-el8-build
```

Comment:
this should be nightly too

Reply:
That is a also a bug in 3.7. Fixing.

Reply:
https://github.com/theforeman/theforeman-rel-eng/pull/223 & https://github.com/theforeman/tool_belt/pull/537